sg_jira/bridge.py

```diff
@@ -10,11 +10,16 @@
 import logging
 import logging.config
 import importlib
+import importlib.util
+import inspect
 from six.moves import urllib
```

Comment:
Also, and only if possible, let's get rid of `six` and replace it for something like

```suggestion
import urllib
```